cmd/cmd.go

```diff
@@ -928,8 +933,13 @@ func generateInteractive(cmd *cobra.Command, opts generateOptions) error {
 			return nil
 		case strings.HasPrefix(line, "/"):
 			args := strings.Fields(line)
-			fmt.Printf("Unknown command '%s'. Type /? for help\n", args[0])
-			continue
+			if multiModal && len(imgFileRegex.FindAllString(args[0], -1)) > 0 {
```

Comment:
I think you can just check if it's a multimodal image, and then call extractFileNames to get each of the filenames on the string. Then you can just look to see if the prefix of the line matches one of the filenames returned by the function. That avoids the global regex and keeps the logic still in the extractFileNames function.

Reply:
clean, made this change